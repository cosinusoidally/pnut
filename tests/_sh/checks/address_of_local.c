// expect_comp_failure
void main() {
  int a;
  int* b = &a;
}
