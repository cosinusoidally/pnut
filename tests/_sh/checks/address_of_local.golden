tests/_sh/checks/address_of_local.c:6:0  comp_rvalue_go: can't take the address of a local variable
